### PR TITLE
Fixed override signature mismatch warnings

### PR DESCRIPTION
**Pull request by thomasfanell**
Reference: [override-signatures](https://github.com/magic-lang/bounty/tree/master/override-signatures)
@marcusnaslund review?

### CODE REVIEW

File: source/ui/win32/Win32DisplayWindow.ooc [hunk 1]
```diff
@@ -15,7 +15,7 @@ version(windows) {
 Win32DisplayWindow: class extends DisplayWindow {
 	_backend: Win32Window
 	init: func (size: IntVector2D, title: String) {
-		super()
+		super(size, title)
```
Comment (marcusnaslund) [inline source/ui/win32/Win32DisplayWindow.ooc]: Interesting. `DisplayWindow` has `init: func (size: IntVector2D, title: String)` which does... nothing. I guess we should rather change its constructor to just be `init: func`. @sebastianbaginski ?

Comment (thomasfanell) [inline source/ui/win32/Win32DisplayWindow.ooc]: I say we should

Comment (sebastianbaginski) [inline source/ui/win32/Win32DisplayWindow.ooc]: It used at least the size last time I checked. Like two months ago.

Comment (sebastianbaginski) [inline source/ui/win32/Win32DisplayWindow.ooc]: Or was it UnixWindow... nevermind, we can remove it now

Comment (thomasfanell) [inline source/ui/win32/Win32DisplayWindow.ooc]: It does in the platform specific implementations

Comment (thomasfanell) [inline source/ui/win32/Win32DisplayWindow.ooc]: I'll remove the parameters where applicable, it's useless to have them there

Comment (sebastianbaginski) [inline source/ui/win32/Win32DisplayWindow.ooc]: OK

**Pull request merged**